specification/security/resource-manager/Microsoft.Security/preview/2022-08-01-preview/securityConnectors.json

```diff
@@ -0,0 +1,1318 @@
+          }
+        },
+        "vaAutoProvisioning": {
+          "type": "object",
+          "description": "The Vulnerability Assessment autoprovisioning configuration",
+          "properties": {
+            "enabled": {
+              "type": "boolean",
+              "description": "Is Vulnerability Assessment auto provisioning enabled"
+            },
+            "configuration": {
+              "type": "object",
+              "description": "configuration for Vulnerability Assessment autoprovisioning",
+              "properties": {
+                "type": {
+                  "type": "string",
+                  "description": "The Vulnerability Assessment solution to be provisioned. Can be either 'TVM' or 'Qualys'",
+                  "enum": [
+                    "Qualys",
+                    "TVM"
+                  ],
+                  "x-ms-enum": {
+                    "name": "type",
+                    "modelAsString": true
+                  }
+                }
+              }
+            }
+          }
+        },
+        "mdeAutoProvisioning": {
+          "type": "object",
+          "description": "The Microsoft Defender for Endpoint autoprovisioning configuration",
+          "properties": {
+            "enabled": {
+              "type": "boolean",
+              "description": "Is Microsoft Defender for Endpoint auto provisioning enabled"
+            },
+            "configuration": {
+              "type": "object",
+              "description": "configuration for Microsoft Defender for Endpoint autoprovisioning",
+              "properties": {}
+            }
+          }
+        },
+        "subPlan": {
+          "type": "object",
+          "description": "configuration for the servers offering subPlan",
+          "properties": {
+            "type": {
+              "type": "string",
+              "description": "The available sub plans",
+              "enum": [
+                "P1",
+                "P2"
+              ],
+              "x-ms-enum": {
+                "name": "subPlan",
+                "modelAsString": true
+              }
+            }
+          }
+        }
+      }
+    },
+    "defenderForDatabasesGcpOffering": {
+      "type": "object",
+      "description": "The Defender for Databases GCP offering configurations",
+      "x-ms-discriminator-value": "DefenderForDatabasesGcp",
+      "allOf": [
+        {
+          "$ref": "#/definitions/cloudOffering"
+        }
+      ],
+      "properties": {
+        "arcAutoProvisioning": {
+          "type": "object",
+          "description": "The ARC autoprovisioning configuration",
+          "properties": {
+            "enabled": {
+              "type": "boolean",
+              "description": "Is arc auto provisioning enabled"
+            },
+            "configuration": {
+              "type": "object",
+              "description": "configuration for arc auto-provisioning",
+              "properties": {}
```

Comment:
>properties [](http://example.com/codeflow?start=15&length=10)

Please provide a definition

Reply:
We want to maintain alignment between all "XXX auto provisioning" variations to follow the structure of { enabled, configuration }
in this case the model expects empty object.
it was already published like this on previous versions (please see mdeAutoProvisioning)

Reply:
we deleted the property "configuration"